src/pbft/pbft.h

```diff
@@ -221,7 +221,11 @@ namespace pbft
       {
         case pbft_message:
           serialized::skip(data, size, sizeof(PbftHeader));
-          message_receiver_base->receive_message((char*)((uint64_t)data), size);
+          if (message_receiver_base->pre_verify(data, size))
+          {
+            message_receiver_base->receive_message(
```

Comment:
Minor: I realise this cast was here already, but its dangerously and unnecessarily removing the const qualifier from data. The called PBFT functions can take a `const` argument, then the cast can be `reinterpret_cast<const char*>(data)`.

Reply:
thanks! I will change that on the pbft side as well!